schemas/Base64.json

```diff
@@ -3,5 +3,5 @@
   "title": "Base64",
   "description": "Base64 string",
   "type": "string",
-  "pattern": "^(?:[A-Za-z0-9+\/]{4}\\n?)*(?:[A-Za-z0-9+\/]{2}==|[A-Za-z0-9+\/]{3}=)$"
+  "pattern": "^(?:[A-Za-z0-9+\/]{4}\\n?)*(?:[A-Za-z0-9+\/]{2}==|[A-Za-z0-9+\/]{3}=)?$"
```

Comment:
I added the `?` here, because it looks like the public keys for ECDSA don't end with `=`. They are base46 according to the spec: https://cyberphone.github.io/openkeystore/resources/docs/jcs.html#publicKey_EC.